### PR TITLE
Some Halloween Costumes with a Crate

### PR DESCRIPTION
**Pull request by ihadtoregisterforthis**
Entirely untested because of some weird compile error on another file.
One mask, one hat, one skelevox skinsuit and a full costume for plasmamen.

Should more costumes be added to the crate?

### CODE REVIEW

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -416,3 +416,9 @@
 	desc = "A conical paper hat which used to be used as a punishment in schools. Misbehaving children had to wear it while standing in a corner. The writing on it says \"DUNCE\"."
 	icon_state = "duncecap"
 	item_state = "duncecap"
+
+/obj/item/clothing/head/snake
+	name = "snake head"
+	desc "Reenact acts of violence against reptiles, or sneak into a swamp unnoticed"
```
Comment (Intigracy) [inline code/modules/clothing/head/misc.dm]: you forgot the =

**Review by Intigracy — CHANGES_REQUESTED** [review]

**Comment by Arthurdentist** [comment]
no pictures?

**Comment by ihadtoregisterforthis** [comment]
They should look like this
an estimate x2

**Comment by FudgePucker** [comment]
:joy: :joy: :joy: :100: :100: :100: :ok_hand: :ok_hand:
Merge when?

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -416,3 +416,9 @@
 	desc = "A conical paper hat which used to be used as a punishment in schools. Misbehaving children had to wear it while standing in a corner. The writing on it says \"DUNCE\"."
 	icon_state = "duncecap"
 	item_state = "duncecap"
+
+/obj/item/clothing/head/snake
+	name = "snake head"
+	desc = "Reenact acts of violence against reptiles, or sneak into a swamp unnoticed"
```
Comment (Exxion) [inline code/modules/clothing/head/misc.dm]: Needs a period.

**Review by Exxion — CHANGES_REQUESTED** [review]

**Comment by ihadtoregisterforthis** [comment]
Fixed both

**Review by Exxion — APPROVED** [review]

**Comment by NigglyWiggly** [comment]
> Emoji mask

George is gettin angry

**Comment by Exxion** [comment]
I like how collabs have the authority to dismiss each other's reviews but not to change their status
God damn it Intigracy approve it before Halloween's over

**Comment by clusterfack** [comment]
You still have the ability to merge it despite that

**Comment by Aurx** [comment]
I really REALLY don't like the emoji mask.  That's not even SS13 memery, that's just generic trash memery.  Why are you suggesting sticking meme shit in for halloween?

**Comment by gbasood** [comment]
>halloween PRs the day of halloween
pls

**Comment by ihadtoregisterforthis** [comment]
The mask is maybe because it was going to happen at some point, and this is a controlled how and when.
Maybe it is to distract players from the course of the codebase so that they can't worry.
Maybe because it is a benchmark of the current culture, and in the future it will remind us of the year when emojis became a voting system and even a movie.

**Comment by Intigracy** [comment]
I'll take out the emoji mask tomorrow but leave the crate in.

**Comment by kol1th** [comment]
😂

**Pull request merged**